pymc3/step_methods/hmc/hmc.py

```diff
@@ -1,12 +1,8 @@
-'''
-Created on Mar 7, 2011
-
-@author: johnsalvatier
-'''
 from ..arraystep import metrop_select, Competence
 from .base_hmc import BaseHMC
 from pymc3.vartypes import discrete_types
 from pymc3.theanof import floatX
```

Comment:
I updated this here -- it is actually a pylint check (`wrong-import-order`) that I can maybe try adding tomorrow, but that is a _ton_ of violations!